examples/plot_loco_minimal.py

```diff
@@ -0,0 +1,143 @@
+"""
+Leave-One-Covariate-Out (LOCO) feature importance with different regression models
+==================================================================================
+
+This example demonstrates how to compare LOCO feature importance :footcite:t:`Williamson_General_2023` across different
+predictive models on the same regression dataset. LOCO is model-agnostic and can be
+applied to any predictive model. Here, we use a linear model, a random forest, a neural
+network, and a support vector machine. We compare the models based on their predictive
+performance (R2 score) and the LOCO feature importance they yield.
+"""
+
+# %%
+import pandas as pd
+from sklearn.datasets import make_regression
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.linear_model import RidgeCV
+from sklearn.model_selection import train_test_split
+from sklearn.neural_network import MLPRegressor
+from sklearn.svm import SVR
+
+from hidimstat import LOCO
+
+# %%
+# Loading and preparing the data
+# ------------------------------
+# We begin by simulating a regression dataset with 10 features, 5 of which
+# are in the support set, meaning they contribute to generating the outcome.
+# The data is then split into training and test sets. These sets are used both to fit
+# the predictive models and within the LOCO procedure, which refits models on subsets
+# of features that exclude the feature of interest.
+
```

Comment:
Wouldn't it be possible to use a public dataset instead ?
Because the data generation process is not really transparent for the reader.

Reply:
OK, I see why: you want a ground truth. But please explain it !